server/datastore/mysql/software_installers.go

```diff
@@ -2305,17 +2307,17 @@ func (ds *Datastore) HasSelfServiceSoftwareInstallers(ctx context.Context, hostP
 	return hasInstallers, nil
 }
 
-func (ds *Datastore) GetSoftwareTitleNameFromExecutionID(ctx context.Context, executionID string) (string, error) {
+func (ds *Datastore) GetDetailsForUninstallFromExecutionID(ctx context.Context, executionID string) (string, bool, error) {
 	stmt := `
-	SELECT st.name
+	SELECT COALESCE(st.name, hsi.software_title_name) name, hsi.self_service
```

Comment:
Do we have `software_titles` rows with a `null` name? I see the definition is `NULL NO` but `Default` is `NULL`. Just wasn't sure we ever ran into that situation.

Reply:
We have this coalesce elsewhere because, while software titles always have a name, software installs can stick around after the associated installer/title is removed. This handles that case.